README.md

```diff
@@ -1,8 +1,761 @@
+
+```bash
+platform ssh
+```
+
+If you are running the command outside of a local copy of the project, you will need to include the `-p` (project) and/or `-e` (environment) flags as well.
+Once you have connected to the container, [logs](https://docs.platform.sh/development/logs.html#container-logs) are available within `/var/log/` for you to investigate.
+
+</details>
+
+
+<details>
+<summary><strong>Rebuilding cache</strong></summary><br/>
+
+You may run into a database error after installing Symfony on your production environment initially.
+To fix, SSH into the application container (`platform ssh`) and rebuild the cache using Symfony console:
+
+```bash
+php bin/console cache:clear
+php bin/console cache:pool:clear cache.redis
+```
+</details>
+
+### Blackfire.io: creating a Continuous Observability Strategy
+
+This template includes a starting [`.blackfire.yml`](.blackfire.yml) file that can be used to enable [Application Performance Monitoring](https://blackfire.io/docs/monitoring-cookbooks/index), [Profiling](https://blackfire.io/docs/profiling-cookbooks/index), [Builds](https://blackfire.io/docs/builds-cookbooks/index) and [Performance Testing](https://blackfire.io/docs/testing-cookbooks/index) on your project. Platform.sh comes with Blackfire pre-installed on application containers, and [setting up requires minimal configuration](https://docs.platform.sh/integrations/observability/blackfire.html).
+
+* [What is Blackfire?](https://blackfire.io/docs/introduction)
+* [Configuring Blackfire.io on a Platform.sh project](https://docs.platform.sh/integrations/observability/blackfire.html)
+* [Blackfire.io Platform.sh documentation](https://blackfire.io/docs/integrations/paas/platformsh)
+* [Profiling Cookbooks](https://blackfire.io/docs/profiling-cookbooks/index)
+* [Monitoring Cookbooks](https://blackfire.io/docs/monitoring-cookbooks/index)
+* [Testing Cookbooks](https://blackfire.io/docs/testing-cookbooks/index)
+* [Using Builds](https://blackfire.io/docs/builds-cookbooks/index)
+* [Configuring Integrations](https://blackfire.io/docs/integrations/index)
+
+
+### Resources
+
+
+- [Symfony](https://symfony.com/)
+- [Symfony on Platform.sh](https://docs.platform.sh/guides/symfony/deploy.html)
+- [Platform.sh PHP documentation](https://docs.platform.sh/languages/php.html)
+
+### Contact
+
+This template is maintained by the Platform.sh Developer Relations team, and they will be notified of all issues and pull requests you open here.
+
+- **Community:** Share your question with the community, or see if it's already been asked on our [Community site](https://community.platform.sh).
+- **Slack:** If you haven't done so already, you can join Platform.sh's [public Slack](https://chat.platform.sh/) channels and ping the `@devrel_team` with any questions.
+
+
+### About Platform.sh
```

Comment:
@flovntp I'd be curious if you have feedback on what should be written here. Is there too much? Too little? Something missing?

Reply:
@chadwcarlson I think it's enough to understand what PSH is and offers. 
Do you see something missing on your POV?